### PR TITLE
Cleanup compiler (webworkify)

### PR DESCRIPTION
**Pull request by axic**

### CODE REVIEW

**Comment by redsquirrel** [comment]
Great to see the continued simplification of app.js 😄

File: src/app/compiler.js [hunk 1]
```diff
@@ -128,21 +128,23 @@ function Compiler(web3, editor, handleGithubCall, outputField, hidingRHP, update
   }
 
   this.loadVersion = function (usingWorker, version, setVersionText) {
+    var url = 'https://ethereum.github.io/solc-bin/bin/' + version;
```
Comment (axic) [inline src/app/compiler.js]: This could also be moved to the caller.

**Comment by axic** [comment]
My end goal is to use `solc-js` instead of the duplicate (twice!) wrapper around soljson.

There's a lot of improvements needed to get there, but the above changes make it way easier.

**Comment by axic** [comment]
@chriseth apply #51 before this as rebasing would be more of a pain otherwise

**Comment by axic** [comment]
Rebased.

@chriseth and @redsquirrel please review these changes

**Comment by redsquirrel** [comment]
Reviewed, and smoke-tested. 👍

**Pull request merged**